contracts/helper.go

```diff
@@ -0,0 +1,59 @@
+package contracts
+
+import (
+	"fmt"
+	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/config"
+)
+
+type DefaultSuiteSetup struct {
+	Config   *config.Config
+	Client   client.BlockchainClient
+	Wallets  client.BlockchainWallets
+	Deployer ContractDeployer
+	Link     LinkToken
+}
+
+var (
+	observationSourceTmpl = `fetch    [type=http method=POST url="%s" requestData="{}"];
+			parse    [type=jsonparse path="data,result"];    
+			fetch -> parse;`
+)
+
+func ObservationSourceSpec(url string) string {
+	return fmt.Sprintf(observationSourceTmpl, url)
+}
+
+func DefaultLocalSetup(initFunc client.BlockchainNetworkInit) (*DefaultSuiteSetup, error) {
```

Comment:
Nice function, but this should go into its own package as it's not contract related.